tests/Doctrine/Tests/DBAL/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -907,4 +907,30 @@ public function testListTableForeignKeysSQLEvaluatesDatabase()
         self::assertContains('bar', $sql);
         self::assertNotContains('DATABASE()', $sql);
     }
+
+    public function testSupportsColumnCollation()
```

Comment:
Can you please add ` : void` for all new test methods?

Reply:
Sure but shouldn't that be changed everywhere? Just doing it here is inconsistent.

Reply:
True, CS is currently not enforced on project level, except checking for some basic rules for new PRs, But soon will be - if we do it like this now, it won't need further fixes in few weeks. 👍